pkg/api/v1alpha1/oidcconfig_webhook.go

```diff
@@ -40,6 +40,14 @@ func (r *OIDCConfig) ValidateUpdate(old runtime.Object) error {
 	if !ok {
 		return apierrors.NewBadRequest(fmt.Sprintf("expected a OIDCConfig but got a %T", old))
 	}
+	oidcconfiglog.Info("Managed vs Workload", "IsManaged()", r.IsManaged())
```

Comment:
do we need this? if so, maybe "Management vs Workload Cluster"? it's a bit confusing and redundant tho since you print the statement after.

Reply:
debugging statement, removing